airflow/utils/db.py

```diff
@@ -882,12 +883,12 @@ def _move_dangling_task_data_to_new_table(session, source_table: "Table", target
     _move_dangling_table(session, source_table, target_table_name, where_clause)
 
 
-def check_task_tables_without_matching_dagruns(session) -> Iterable[str]:
+def check_task_tables_without_matching_dagruns(session: Session) -> Iterable[str]:
     import sqlalchemy.schema
     from sqlalchemy import and_, outerjoin
 
     metadata = sqlalchemy.schema.MetaData(session.bind)
-    models_to_dagrun = [TaskInstance, TaskReschedule]
+    models_to_dagrun: List[Any] = [TaskInstance, TaskReschedule]
```

Comment:
Technically we can type this (what we want is the `__tablename__` attribute from SQLAlchemy), but SQLAlchemy is so dynamic it does not really help much, so I don’t really want to bother.

Reply:
👍🏻 Also, as this isn't called anywhere from outside of this module it doesn't impact the typing of the rest of the code base.